.gitlab-ci.yml

```diff
@@ -308,3 +308,28 @@ pages:
   only:
     - master
 
+package:appimage-client:
+  stage: package
+  image: appimagecrafters/appimage-builder
+  dependencies:
+    - build:ubuntu-16.04
```

Comment:
why so outdated?

Reply:
I didn't wanted to add another build so I recycled one. It's the newest ubuntu build in the gitlab ci.
I agree it should be updated but that's just not the goal of this PR

Reply:
14.04 should be removed in favor of 18.04 then.
Please do that in a separate commit so it can be kept apart on merge.

Reply:
Did that.
Build is just like in ubuntu:16-04